main.go

```diff
@@ -219,6 +219,17 @@ func main() {
 
 	restConfig := ctrl.GetConfigOrDie()
 	restConfig.UserAgent = remote.DefaultClusterAPIUserAgent("cluster-api-controller-manager")
+
+	minVer := version.MinimumKubernetesVersion
```

Comment:
In general lgtm but I think this will break our mink8s 1.20 e2e test. But _I think_ we cam disable the feature flag there (we just have to modify the job in test-infra)